engine/swarm/admin_guide.md

```diff
@@ -267,11 +296,16 @@ balance across the swarm. When new tasks start, or when a node with running
 tasks becomes unavailable, those tasks are given to less busy nodes. The goal
 is eventual balance, with minimal disruption to the end user.
 
-If you are concerned about an even balance of load and don't mind disrupting
-running tasks, you can force your swarm to re-balance by temporarily scaling
-the service upward.
+In Docker 1.13 and higher, you can use the `--force` or `-f` flag with the
+`docker service update` command to force the service to redistribute its tasks
+across the available worker nodes. This will cause the service tasks to restart.
+Client applications may be disrupted. If you have specified an `--update-delay`
+value for your service, the service will use a rolling restart.
 
-Use `docker service inspect --pretty <servicename>` to see the configured scale
+If you use an earlier version and you want to achieve an even balance of load
+across workers and don't mind disrupting running tasks, you can force your swarm
+to re-balance by temporarily scaling the service upward. Use
```

Comment:
This is the admin guide that lives on docker.com. It is versioned in a way (you can get to older versions), but we also need to be careful because lots of people just Google. When a feature is brand new, we need to call out when you can expect it to work. Otherwise, support will get lots of calls from people on earlier versions trying to use commands.